test/parallel/test-buffer-isencoding.js

```diff
@@ -17,7 +17,8 @@ const assert = require('assert');
     assert.strictEqual(Buffer.isEncoding(enc), true);
   });
 
-[ 'utf9',
+[ '',
+  'utf9',
```

Comment:
@sam-github So i think the problem we are facing now is that those APIs which use encodings identify **invalid encodings** inconsistently, and we can't tell whose behavior is actually right ?